docs/send-to-device.rst

```diff
@@ -72,23 +72,6 @@ The Jinja macro supports parameters as follows (* indicates a required parameter
 +----------------------+------------------------------------------------------------------------+----------------------+--------------------------------------------------------------------+
 |    legal_note_email  | Provides a custom legal note for email use.                            | Localizable String.  | _('The intended recipient of the email must have consented.')      |
 +----------------------+------------------------------------------------------------------------+----------------------+--------------------------------------------------------------------+
-|    legal_note_sms    | Provides a custom legal note for SMS or email.                         | Localizable string   | _('SMS service available in select countries only.')               |
-+----------------------+------------------------------------------------------------------------+----------------------+--------------------------------------------------------------------+
 |    spinner_color     | Hex color for the form spinner. Defaults to '#000'.                    | String               | '#fff'                                                             |
 +----------------------+------------------------------------------------------------------------+----------------------+--------------------------------------------------------------------+
 
-
-Geolocation Callback
```

Comment:
Are we sure we won't need the Geo Callback anymore? We do still sometimes restrict mobile products to certain countries.

Reply:
Right now the only restrictions we have are to only show the email form to locales which have translated emails and that's handled in the python. Since this is dead code now I'd err on the side of removing it but if there's a specific use case we know is coming I can save it.

Reply:
Agree, we can still make an XHR request to the geo URL for other reasons. I don't think it's worth keeping this in the S2D widget if it's functionality no longer related to the form 👍